fluid/DeepASR/data_utils/load_data.py

```diff
@@ -0,0 +1,183 @@
+#by zhxfl 2018.01.24
+""" @package docstring
+load speech data from disk
+"""
+
+import random
+import Queue
+import numpy
+import struct
+import data_utils.trans_mean_variance_norm as trans_mean_variance_norm
+import data_utils.trans_add_delta as trans_add_delta
+
+g_lblock = []
+g_que_sample = Queue.Queue()
+g_nframe_dim = 120 * 11
+g_nstart_block_idx = 0
+g_nload_block_num = 1
+g_ndrop_frame_len = 256
+
+class OneBlock(object):
+    """ Documentation for a class.
+        struct for one block :
+        contain label, label desc, feature, feature_desc
+    """
+    def __init__(self):
+        """The constructor."""
+        self.label = ""
+        self.label_desc = ""
+        self.feature = ""
+        self.feature_desc = ""
+
+def set_trans(ltrans):
```

Comment:
Please consider wrap data processing related methods into a class and avoid using global variables.